R/get_data.R

```diff
@@ -32,21 +32,24 @@ get_data <- function(psid, start_date = NULL, end_date = NULL, analyte = NULL) {
   }
 
   if(!is.character(psid)) {
-    stop("Argument `psid` must be of class `character`. To view a psid lookup table, see the built-in object `psid_df`.",
+    stop("Argument `psid` must be of class `character`.
+         To view a table of water systems lookup table,
+         see the built-in object `water_systems`.",
          call. = FALSE)
   }
+
   url = paste0('https://sdwis.s3.us-west-1.amazonaws.com/', psid, '.csv.gz')
   d   = data.table::fread(url)
 
   # if present, filter by start and end dates and analytes
   if(!is.null(start_date)){
-    d <- dplyr::filter(d, SAMP_DATE >= start_date)
+    d <- d[d$SAMP_DATE >= start_date, ]
```

Comment:
could you pick a more informative variable name than `d`?

Reply:
I'm thinking `s3_dat`. Also writing some tests for get_data(), and will add these to the PR.

Reply:
I've updated d to s3_dat, and added some more to the function, see commits below.